app/src/main/java/com/example/mykotlincalculator21/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.mykotlincalculator21.mvp.presenter
+
+import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
+
+class CalculatorPresenter(model: CalculatorContract.CalculatorModelContract, view: CalculatorContract.CalculatorViewContract): CalculatorContract.CalculatorPresenterContract {
+
+    private val model: CalculatorContract.CalculatorModelContract
```

Comment:
you can remove these variables and add private val before each one in the class constructor

class CalculatorPresenter(private val model: CalculatorContract.CalculatorModelContract, private val view: CalculatorContract.CalculatorViewContract)

Reply:
Done. Thx